src/modules/layout/components/TransparentHeader.js

```diff
@@ -79,17 +84,17 @@ class TransparentHeader extends React.PureComponent<PropsType> {
   }
 
   render () {
-    const { t } = this.props
+    const { theme, withMenu, float, t } = this.props
 
     return (
-      <BoxShadow theme={this.props.theme}>
+      <BoxShadow theme={theme} float={float}>
         <Horizontal>
           <HorizontalLeft>
             <HeaderBackButton onPress={this.goBack} />
           </HorizontalLeft>
-          <MaterialHeaderButtons>
+          { withMenu && <MaterialHeaderButtons>
             <Item title={t('share')} show='never' onPress={this.onShare} />
-          </MaterialHeaderButtons>
+          </MaterialHeaderButtons> }
```

Comment:
Do we really need the `withMenu` property? We could instead just display the share button if we have `url` in `this.props.navigation.state.params`.
`withMenu` sounds quite abstract for it only deciding to show a `share`-button or not (quite specific).